.changeset/twenty-meals-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": minor
+---
+
+Added new lint rule `noShadow`, a port of eslint's `no-shadow`. Contributed by @dyc3.
```

Comment:
Couple of notes:
- we usually add a link to the new rule 
- we should explain what we rule does with a small example 
- we don't need the "contributed by" anymore, the changesets does it for us